### PR TITLE
Enable GKEY coinGeckoId

### PR DESCRIPTION
**Pull request by Rizbe**

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -468,7 +468,7 @@ const chainInfos = (
           coinMinimalDenom:
             "cw20:juno1gz8cf86zr4vw9cjcyyv432vgdaecvr9n254d3uwwkx9rermekddsxzageh:GKEY",
           coinDecimals: 6,
-          // coinGeckoId: "pool:gkey",
+          coinGeckoId: "pool:gkey",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:471]: this id is not defined (but you can do so in /config/price.ts). No point in uncommenting this without it defined.

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
please first define this id

**Comment by Rizbe** [comment]
Updated the PR

I assume once this is merged we will get gkey price data using: https://api-osmosis.imperator.co/tokens/v2/price/gkey 

Currently it says: `{"message":"Oops! Something went wrong with gkey ..."}`

**Review by JeremyParish69 — APPROVED** [review]
LGTM

**Pull request merged**